### PR TITLE
Geoserver 2.7.2

### PR DESCRIPTION
**Pull request by waynegraham**
updates geoserver to 2.7.2

### CODE REVIEW

File: Library/Formula/geoserver.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
+require 'formula'
+
```
Comment (bfontaine) [inline Library/Formula/geoserver.rb]: Don’t add these lines, thanks.

File: Library/Formula/geoserver.rb [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 class Geoserver < Formula
   desc "Java server to share and edit geospatial data"
-  homepage "http://geoserver.org/"
-  url "https://downloads.sourceforge.net/project/geoserver/GeoServer/2.7.1.1/geoserver-2.7.1.1-bin.zip"
-  sha256 "4c584ae1e586736533e3a4bd9969eb0d180ec683cf79f2aff8512075b742da60"
+  homepage 'http://geoserver.org/'
+  url 'https://downloads.sourceforge.net/project/geoserver/GeoServer/2.7.2/geoserver-2.7.2-bin.zip'
+  sha256 '2fffa49bc46743673e4fd678873ff2891e1a31471a10758aa0603c30fea2b1eb'
```
Comment (bfontaine) [inline Library/Formula/geoserver.rb]: Please keep the double quotes.

File: Library/Formula/geoserver.rb [hunk 3]
```diff
@@ -19,13 +21,12 @@ def install
...
       geoserver path/to/data/dir
-
     See the Geoserver homepage for more setup information:
       brew home geoserver
     EOS
   end
 
   test do
-    assert_match /geoserver path/, shell_output("#{bin}/geoserver")
+    assert_match /geoserver path/, shell_outout("#{bin}/geoserver")
```
Comment (bfontaine) [inline Library/Formula/geoserver.rb]: Typo here, `outout`→`output`

**Comment by DomT4** [comment]
Closing as dead. Feel free to resubmit with the comments from Baptiste addressed as desired. Thanks for the submission to Homebrew.

**Pull request closed without merging**